repos/c2sm/packages/icon/package.py

```diff
@@ -31,20 +31,20 @@ class Icon(AutotoolsPackage, CudaPackage):
 
     homepage = 'https://code.mpimet.mpg.de/projects/iconpublic'
     url = 'https://gitlab.dkrz.de/icon/icon/-/archive/icon-2.6.6/icon-icon-2.6.6.tar.gz'
-    git = 'ssh://git@gitlab.dkrz.de/icon/icon.git'
+    git = 'git@gitlab.dkrz.de:icon/icon.git'
 
     version('develop', submodules=True)
     version('2.6.6', tag='icon-2.6.6', submodules=True)
     version('exclaim-master',
             branch='master',
-            git='ssh://git@github.com/C2SM/icon-exclaim.git',
+            git='git@github.com:C2SM/icon-exclaim.git',
             submodules=True)
     version('exclaim',
-            branch='icon-dsl',
-            git='ssh://git@github.com/C2SM/icon-exclaim.git',
+            branch='icon-dsl-spack',
```

Comment:
```suggestion
            branch='icon-dsl',
```
Does this need reverting to icon-dsl?